packages/ai/core/util/index.ts

```diff
@@ -1,4 +1,4 @@
-export { generateId } from '@ai-sdk/provider-utils';
+export * from '@ai-sdk/provider-utils';
```

Comment:
avoid export `*` - we need to control what we re-export

Reply:
aha, will make sure to do in future 🙏